bucket/windterm.json

```diff
@@ -6,6 +6,12 @@
     "suggest": {
         "vcredist2022": "extras/vcredist2022"
     },
+    "notes": [
+        "From version 2.4.0, WindTerm changed the way how the connection profiles are stored.",
+        "During the first run of version 2.4.0 or higher, please, choose the first option for 'profile directory', so the profiles will be stored along the application files.",
+        "Then copy the old profiles directory, stored under 'persist/WindTerm/profiles', over the newly created folder under 'persist/WindTerm/.wind/profiles'. After that you can delete the old 'profiles' directory.",
```

Comment:
Can you automate this process? Have a look at vscode's manifest to see how we handle data migrations: https://github.com/ScoopInstaller/Extras/blob/f318f78fc56137877a573495c6f56b805f52090f/bucket/vscode.json#L48